robolectric/src/main/java/org/robolectric/internal/ParallelUniverse.java

```diff
@@ -139,28 +142,16 @@ public void setUpApplicationState(Method method, TestLifecycle testLifecycle, Re
 
   @Override
   public void tearDownApplication() {
+    // We would like to null this out (which would call reset) but that can kill async threads
```

Comment:
Picasso broke this in our tests (yeah I know), however I think @erd just fixed it: https://github.com/robolectric/robolectric/commit/b50b3d86fd168208ee6065385f89b440591733b6